mackerel/check.go

```diff
@@ -1,41 +1,16 @@
 package mackerel
 
 import (
-	"encoding/json"
-
 	"github.com/mackerelio/mackerel-agent/checks"
+	mkr "github.com/mackerelio/mackerel-client-go"
 )
 
-type monitoringChecksPayload struct {
-	Reports []*checkReport `json:"reports"`
-}
-
-type checkReport struct {
-	Source               monitorTargetHost `json:"source"`
-	Name                 string            `json:"name"`
-	Status               checks.Status     `json:"status"`
-	Message              string            `json:"message"`
-	OccurredAt           Time              `json:"occurredAt"`
-	NotificationInterval *int32            `json:"notificationInterval,omitempty"`
-	MaxCheckAttempts     *int32            `json:"maxCheckAttempts,omitempty"`
-}
-
-type monitorTargetHost struct {
-	HostID string
-}
-
-// MarshalJSON implements json.Marshaler.
-func (h monitorTargetHost) MarshalJSON() ([]byte, error) {
-	return json.Marshal(map[string]string{
-		"type":   "host",
-		"hostId": h.HostID,
-	})
-}
+const NotificationIntervalFallback = 10
```

Comment:
`make lint` fails. Please add some comments.

Reply:
sorry, I resolved it.